Header header
Kinematics robot_kinematics
Kinematics fly_kinematics
Stopped robot_stopped
Stopped fly_stopped
InBounds robot_in_bounds
InBounds fly_in_bounds